### PR TITLE
fix(Explore): fixes broken layout of tooltips

### LINKED ISSUE #12689: [explore] Dropdowns tooltip icon jumping when mouse is on label
## Screenshot

https://user-images.githubusercontent.com/25153919/105512024-e7645f80-5cd0-11eb-80b6-e52abcacb445.mov

## Description

Tooltip icon blinks and jump when I put mouse on these fields.

Steps to reproduce:
Add chart with multiline viz type
Go to Y Axis Left section and put mouse on dropdowns

I would expect to not see this jumping - no action when mouse is on text/field and tooltip displayed when mouse on tooltip icon.

### PR DESCRIPTION
**Pull request by rusackas**
### SUMMARY

Fixes an issue where Explore control panels with long label text cause funky line-wrapping on mouseover. This was due to the `span` containing the tooltip(s) appearing inline, and getting to wide. This PR makes the tooltips absolutely positioned, so that they can't cause a line wrap. They might hang a little too far over to the right, but it's an improvement.

As bycatch, this moves a bunch of CSS from LESS files to the component itself, using Emotion and theme variables.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before:

https://user-images.githubusercontent.com/67837651/118920442-4c306f00-b8eb-11eb-8756-11c88bb52dcb.mov

After:
fixed

### ADDITIONAL INFORMATION

- [ ] Has associated issue: Fixes #12689
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/explore/components/ControlPanelsContainer.tsx [hunk 1]
```diff
@@ -211,7 +212,36 @@ export class ControlPanelsContainer extends React.Component<ControlPanelsContain
...
+
+          .panel-body {
+            margin-left: ${theme.gridUnit * 4}px;
+            padding-bottom: 0px;
+          }
+
+          .control-label {
+            margin-bottom: 0px;
+            position: relative;
+            span + span {
```
Comment (villebro) [inline superset-frontend/src/explore/components/ControlPanelsContainer.tsx:232]: I was wondering if we should add class names to these and target those?

**Review by villebro — COMMENTED** [review]
I'm still getting the wrapping problem: 
tooltip

**Comment by rusackas** [comment]
> I'm still getting the wrapping problem:
> tooltip

Ugh... it fixes SOME of them. Some are coming from a different component, and will require additional styles. More commits landing here soon.

File: superset-frontend/src/explore/components/ControlPanelsContainer.tsx (hunk 1 above)
Comment (rusackas) [reply superset-frontend/src/explore/components/ControlPanelsContainer.tsx]: I went ahead and moved these styles to a better location. I'll sleep better now, thanks for the suggestion.

**Comment by rusackas** [comment]
@villebro You uncovered something gross... this PR only fixes SOME of the controls. 

A bunch of controls use `ControlHeader` from `superset`, and some use `ControlHeader` from  `superset-ui` 🤦 

Proposed approach: 
1) Merge this (it fixes several)
2) Merge a PR on superset-ui (which I'll open soon) that does the same thing with the `ControlHeader` on that side
3) Bump in the `@superset-ui/chart-controls` package, and use that updated `ControlHeader` from `superset-ui` everywhere, and delete the duplicate one (this one!) from Superset.

**Comment by rusackas** [comment]
Step 2 of the plan opened here: https://github.com/apache-superset/superset-ui/pull/1107

File: superset-frontend/src/explore/components/ControlPanelsContainer.tsx (hunk 1 above)
Comment (rusackas) [reply superset-frontend/src/explore/components/ControlPanelsContainer.tsx]: I think they're a little more solid now too... see if you can break it 🤞

File: superset-frontend/src/explore/components/ControlHeader.jsx [hunk 2]
```diff
@@ -49,7 +49,16 @@ export default class ControlHeader extends React.Component {
   renderOptionalIcons() {
     if (this.props.hovered) {
       return (
-        <span>
+        <span
+          css={theme => css`
```
Comment (michael-s-molina) [inline superset-frontend/src/explore/components/ControlHeader.jsx:53]: `css` 👀 ... 🤣

**Review by suddjian — APPROVED** [review]
LGTM in light of ***the plan™***

File: superset-frontend/src/explore/components/ControlHeader.jsx (hunk 2 above)
Comment (rusackas) [reply superset-frontend/src/explore/components/ControlHeader.jsx:53]: The debate of the ages! I'm actually working on a doc to specify use cases for each, and try to arrive at some sort of best practices. I'll share it when it's fully baked.

**Pull request merged**